README.md

```diff
@@ -1,5 +1,29 @@
 # Process CSV reports
 
+The processing script `process_report.py` takes in the monthly invoices and goes through several processing and exporting steps:
+1. Combines all provided invoices (more info below)
+2. Obtain each PI's institution name
+3. Exports the list of non-billable projects (more info below)
+4. Apply the New PI credit
+5. Exports the billable projects as one csv, by PI, for HU PIs only, for HU and BU PIs, and for projects with Lenovo SU Types
+
+The CSV invoices must at least contain the following headers:
+- Invoice Month
+- Project - Allocation
+- Manager (PI)
+- Institution
+- SU Hours (GBhr or SUhr)
+- SU Type
+- Cost
+
+```
+usage: process_report.py [-h] --pi-file PI_FILE --projects-file PROJECTS_FILE --timed-projects-file TIMED_PROJECTS_FILE [--output-file OUTPUT_FILE]
+                         [--output-folder OUTPUT_FOLDER] [--HU-invoice-file HU_INVOICE_FILE] [--HU-BU-invoice-file HU_BU_INVOICE_FILE] [--old-pi-file OLD_PI_FILE]
```

Comment:
can you add how a sample OLD_PI_FILE would look like? 

The non-billable section of this document shows sample files for pi.txt, projects.txt etc.

Reply:
Done